### ISSUE TITLE
thinc.extra.search.MaxViolation on 6.8.2

### ISSUE THREAD

**Issue by chssch** [explosion/thinc#38]
Debian GNU/Linux 8
Python: 3.6
Thinc: 6.8.2

I get this error while loading spacy lang or e.g. `NeuralEntityRecognizer`, it works with thinc 6.8.1 though:

```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-3-d57cdd72c72b> in <module>()
----> 1 from spacy.pipeline import NeuralEntityRecognizer

/usr/local/lib/python3.6/site-packages/spacy/pipeline.pyx in init spacy.pipeline (spacy/pipeline.cpp:46029)()
----> 1 # cython: infer_types=True
      2 # cython: profile=True
      3 # coding: utf8
      4 from __future__ import unicode_literals
      5 

/usr/local/lib/python3.6/site-packages/spacy/syntax/arc_eager.pxd in init spacy.syntax.parser (spacy/syntax/parser.cpp:39076)()
     10 
     11 
---> 12 cdef class ArcEager(TransitionSystem):
     13     pass
     14 

search.pxd in init spacy.syntax.arc_eager (spacy/syntax/arc_eager.cpp:21709)()

ValueError: thinc.extra.search.MaxViolation has the wrong size, try recompiling
```

**Comment by honnibal** [comment]
Try uninstalling, then reinstalling with `pip install spacy-nightly --no-cache-dir`?

**Comment by chssch** [comment]
Thanks @honnibal , unfortunately that didn't help, even if I do a full reinstall of the whole container (docker) I get this:

```
File "py.py", line 2, in <module>
    spacy.load("en")
  File "/usr/local/lib/python3.6/site-packages/spacy/__init__.py", line 13, in load
    return util.load_model(name, **overrides)
  File "/usr/local/lib/python3.6/site-packages/spacy/util.py", line 103, in load_model
    return load_model_from_link(name, **overrides)
  File "/usr/local/lib/python3.6/site-packages/spacy/util.py", line 122, in load_model_from_link
    return cls.load(**overrides)
  File "/usr/local/lib/python3.6/site-packages/spacy/data/en/__init__.py", line 12, in load
    return load_model_from_init_py(__file__, **overrides)
  File "/usr/local/lib/python3.6/site-packages/spacy/util.py", line 155, in load_model_from_init_py
    return load_model_from_path(data_path, meta, **overrides)
  File "/usr/local/lib/python3.6/site-packages/spacy/util.py", line 136, in load_model_from_path
    cls = get_lang_class(meta['lang'])
  File "/usr/local/lib/python3.6/site-packages/spacy/util.py", line 41, in get_lang_class
    module = importlib.import_module('.lang.%s' % lang, 'spacy')
  File "/usr/local/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 978, in _gcd_import
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 655, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 205, in _call_with_frames_removed
  File "/usr/local/lib/python3.6/site-packages/spacy/lang/en/__init__.py", line 15, in <module>
    from ...language import Language
  File "/usr/local/lib/python3.6/site-packages/spacy/language.py", line 19, in <module>
    from .syntax.parser import get_templates
  File "arc_eager.pxd", line 12, in init spacy.syntax.parser
  File "search.pxd", line 72, in init spacy.syntax.arc_eager
ValueError: thinc.extra.search.MaxViolation has the wrong size, try recompiling. Expected 104, got 128
```

## Info about spaCy

* **spaCy version:** 2.0.0a14
* **Platform:** Linux-4.9.27-moby-x86_64-with-debian-8.9
* **Python version:** 3.6.2
* **Models:** en


## Info about model en

* **lang:** en
* **pipeline:** ['tensorizer', 'tagger', 'parser', 'ner']
* **name:** core_web_sm
* **license:** CC BY-SA
* **author:** Explosion AI
* **url:** https://explosion.ai
* **version:** 2.0.0a4
* **spacy_version:** >=2.0.0a14
* **parent_package:** spacy-nightly
* **email:** contact@explosion.ai
* **description:** English multi-task CNN trained on OntoNotes, with GloVe vectors trained on common crawl. Assigns word vectors, context-specific token vectors, POS tags, dependency parse and named entities.

Lmk, if I can check something else.

**Comment by thomasopsomer** [comment]
I'm having the same issue on macOs. I tried a fresh install, in a new virtual env, in python 2 or 3. As well as a `--no-cache-dir`. But still doesn't work. (but works with thinc==6.8.1)

**Comment by honnibal** [comment]
Should all be fixed in Thinc 6.9. Thanks for your patience!
